### PR TITLE
Add codecov for testing

### PR DESCRIPTION
**Pull request by ulya-tkch**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,9 +18,11 @@ jobs:
       - name: Setup python environment
         run: ./.github/scripts/setup_python_env.sh
         shell: bash
+      - name: Install working dir package
```
Comment (sanjanag) [inline .github/workflows/ci.yml:21]: `./.github/scripts/setup_python_env.sh` installs the package

Comment (ulya-tkch) [reply .github/workflows/ci.yml:21]: Without this there is a file path mismatch and 0% coverage score. Installing the package in the working directory using pip's editable mode avoids it.

**Review by sanjanag — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jwmueller) [reply .github/workflows/ci.yml]: what is the recommended way for developers to install this package?

File: .github/scripts/setup_python_env.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 python -m pip install --upgrade pip
-pip install .
+pip install -e .
```
Comment (sanjanag) [inline .github/scripts/setup_python_env.sh:4]:
```suggestion
pip install .
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -19,8 +19,8 @@ jobs:
         run: ./.github/scripts/setup_python_env.sh
         shell: bash
       - name: Test with coverage
-        run: pytest --verbose --cov=cleanvision/ --cov-config .coveragerc --cov-report=xml
-      - uses: codecov/codecov-action@v2
+        run: pytest --verbose --cov=src/cleanvision/ --cov-config .coveragerc --cov-report=xml tests/
```
Comment (sanjanag) [inline .github/workflows/ci.yml:22]:
```suggestion
        run: pytest --verbose --cov=cleanvision --cov-config .coveragerc --cov-report=xml tests/
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sanjanag) [reply .github/workflows/ci.yml]: I figured if you specify the package_name instead of path it works

**Review by sanjanag — APPROVED** [review]

**Pull request merged**